SEED-platform/seed
# Move program setup to modal on insights pages

To prepare for the portfolio (cross cycle) summary coming soon to the insights tab, the program setup page was moved to a modal accessible from both the property insights and program insight pages. 

<img width="1473" alt="Screen Shot 2024-01-08 at 4 31 10 PM" src="https://github.com/SEED-platform/seed/assets/2205659/f02caa85-4e29-4ced-bdfb-bf0552d50718">

<img width="1491" alt="Screen Shot 2024-01-08 at 4 31 21 PM" src="https://github.com/SEED-platform/seed/assets/2205659/2b47d1d1-1e73-4245-a226-44ed54dbb5d4">

<img width="256" alt="Screen Shot 2024-01-08 at 4 30 53 PM" src="https://github.com/SEED-platform/seed/assets/2205659/c908c70c-9ed2-49de-ad75-22ed2c20f18f">

To test:
ensure that you can still CRUD programs from both the insights pages and that the page behaves property when dismissing the modal (for example, if you delete the program that is currently displayed on the page, it should select another program to display)
